test/echo/gradle.properties_example

```diff
@@ -0,0 +1 @@
+org.gradle.java.home=/Library/Java/JavaVirtualMachines/jdk1.7.0_55.jdk/Contents/Home/
```

Comment:
Your test app should be inline with the gradle version (3.5 and jdk8). It also needs to listen on port 8080